eng/targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -1,56 +1,20 @@
+          BeforeTargets="GenerateBindingRedirects"
+          DependsOnTargets="ResolveAssemblyReferences"
+          Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <ItemGroup>
-      <!-- The version 0.0.0.0 in these references doesn't actually matter; the generated versions will always generate from 0.0.0.0-->
-      <SuggestedBindingRedirects Include="Microsoft.CodeAnalysis, Version=0.0.0.0, Culture=neutral, PublicKeyToken=$(PublicKeyToken)">
-        <MaxVersion>$(AssemblyVersion)</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="Microsoft.CodeAnalysis.CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=$(PublicKeyToken)">
-        <MaxVersion>$(AssemblyVersion)</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="Microsoft.CodeAnalysis.VisualBasic, Version=0.0.0.0, Culture=neutral, PublicKeyToken=$(PublicKeyToken)">
-        <MaxVersion>$(AssemblyVersion)</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Buffers, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51">
-        <MaxVersion>4.0.2.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Collections.Immutable, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-        <MaxVersion>1.2.3.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Memory, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51">
-        <MaxVersion>4.0.1.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Numerics.Vectors, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51">
-        <MaxVersion>4.1.3.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Reflection.Metadata, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-        <MaxVersion>1.4.3.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Runtime.CompilerServices.Unsafe, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-        <MaxVersion>4.0.4.1</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Text.Encoding.CodePages, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-        <MaxVersion>4.1.1.0</MaxVersion>
-      </SuggestedBindingRedirects>
-      <SuggestedBindingRedirects Include="System.Threading.Tasks.Extensions, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51">
-        <MaxVersion>4.2.0.0</MaxVersion>
-      </SuggestedBindingRedirects>
+      <SuggestedBindingRedirects Include="@(ReferencePath->'%(FusionName)')" MaxVersion="%(ReferencePath.Version)" KeepMetadata="-none-" Condition="'%(ReferencePath.FrameworkFile)' != 'true'" />
```

Comment:
Can you share a diff of config file before and after this change?

Reply:
It's the same as the one in previous commit:
https://github.com/dotnet/roslyn/pull/33993/commits/6e54564f7e4c5e6404f2a42eedf9b50afbb79586

Reply:
I mean the values are.